scss/components/_menu.scss

```diff
@@ -231,6 +231,15 @@ $menu-icon-spacing: 0.25rem !default;
     // Orientation
     @include menu-direction(horizontal);
 
+    // Even-width
+    &.expanded {
+      @include menu-expand;
+
+      > li:first-child:last-child {
+        width: 100%;
+      }
```

Comment:
Moved to the mixin. The idea was to keep the code the same, just move it around and fix the specificity issues